index.d.ts

```diff
@@ -0,0 +1,84 @@
+declare module 'react-native-google-cast' {
+  import * as React from 'react'
+  import { EventEmitter } from 'react-native'
+
+  export type CastDevice = {
+    id: string
+    version: string
+    name: string
+    model: string
+  }
+
+  export type CastState =
+    | 'NoDevicesAvailable'
+    | 'NotConnected'
+    | 'Connecting'
+    | 'Connected'
+
+  export type CastOptions = Partial<{
```

Comment:
Please don't use `Partial`.

`mediaUrl` is required, all the others are optional.

Reply:
corrected, is Partial still ok for TextTrackStyle? (which is not in this PR but in https://github.com/react-native-google-cast/react-native-google-cast/pull/127)

Reply:
I'd use Partial (or Required/Omit/Pick) only to modify an external type.

In the case here, it's much clearer to use `?` for optional props. Also, with Partial, you can't make a field required.

Reply:
got it, switching away from partial. TextTrackStyle that is used in #127 has every prop optional so it is marked here that way to match.